src/ol/object.js

```diff
@@ -8,6 +9,33 @@ goog.require('ol.Observable');
 
 
 /**
+ * Polyfill for Object.assign().  Assigns enumerable and own properties from
+ * one or more source objects to a target object.
```

Comment:
This is from https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill, right? Maybe we should link there?